cli/list_cmd.ml

```diff
@@ -0,0 +1,138 @@
+open Import
+open Duniverse
+
+type t = {
+  name : string;
+  version : string;
+  loc : string;
+  pinned : bool;
+  descr : string option;
+}
+
+(* Assumes max_len was correctly computed, otherwise the input string might be
+   truncated. *)
+let pad s max_len =
+  let len = String.length s in
+  String.init max_len ~f:(fun i -> if i < len then s.[i] else ' ')
+
+(* FIXME: replace this once we remember the remotes used during lock. *)
```

Comment:
If this can be turned into a pointer to an issue that's more probability that it's going to be addressed :)

Reply:
I'll simply remove this one as I'm not sure we'll ever remember remotes.